charts/external-secrets/values.yaml

```diff
@@ -1,2 +1,14 @@
 external-secrets:
   installCRDs: true
+
+secretsSource:
+  enabled: true
+  url: ssh://git@github.com/owner/reponame    # for the CRs (secret store, secrets)
```

Comment:
`# for the CRs (secret store, secrets)` when do you think we would have secrets manifests in the repo?

Reply:
This is the url for the repository that has the SecretStore and ExternalSecrets CRs, It could be in another repo than the profile